docs/api/baseplate/clients/cassandra.rst

```diff
@@ -0,0 +1,62 @@
+``baseplate.clients.cassandra``
+===============================
+
+.. automodule:: baseplate.clients.cassandra
+
+This integration supports both the base Python Cassandra driver and the
```

Comment:
This may be a 🔕 , but would it be worth a sentence or two that says something like "Cassandra is a X. This class is a client that does Y", with a link to Apache Cassandra. Rationale being that some engineers may not have seen Cassandra prior to their stay at reddit.